docs/sources/reference/components/otelcol.exporter.awss3.md

```diff
@@ -0,0 +1,190 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/components/otelcol.exporter.awss3/
+description: Learn about otelcol.exporter.awss3
+title: otelcol.exporter.awss3
+---
+
+<span class="badge docs-labels__stage docs-labels__item">Experimental</span>
+
+# otelcol.exporter.awss3
+
+{{< docs/shared lookup="stability/experimental.md" source="alloy" version="<ALLOY_VERSION>" >}}
+
+`otelcol.exporter.awss3` accepts telemetry data from other `otelcol` components
+and writes them to an AWS S3 Bucket.
+
+> **NOTE**: `otelcol.exporter.awss3` is a wrapper over the upstream
+> OpenTelemetry Collector Contrib `awss3` exporter. Bug reports or feature requests will
+> be redirected to the upstream repository, if necessary.
+
+Multiple `otelcol.exporter.awss3` components can be specified by giving them
+different labels.
+
+## Usage
+
+```alloy
+otelcol.exporter.awss3 "LABEL" {
+  s3_uploader {
+    region = "REGION"
+    s3_bucket = "BUCKET_NAME"
+    s3_prefix = "PREFIX"
+  }
+}
+```
+
+## Arguments
+
+`otelcol.exporter.awss3` supports the following arguments:
+
+Name       | Type       | Description                                      | Default | Required
+-----------|------------|--------------------------------------------------|---------|---------
+`encoding` | `string`   | Encoding extension to use to marshal data. Overrides the `marshaler` configuration option if set. | `""`  | no
+`encoding_file_ext` | `string` | file format extension suffix when using the `encoding` configuration option. May be left empty for no suffix to be appended. | `""` | no
```

Comment:
```suggestion
`encoding_file_ext` | `string` | File format extension suffix when using the `encoding` configuration option. It can be left empty if a suffix shouldn't be appended. | `""` | no
```